### DISCUSSION TITLE
上传什么文件都联不上服务器

### DISCUSSION THREAD

**Discussion by ingale726** [houtianze/bypy#667] [Q&A]
centos7.9
python3.10

<E> [13:16:13] Waiting 10 seconds before retrying...
<E> [13:16:23] Request Try #2 / 5
<E> [13:16:23] Waiting 20 seconds before retrying...
<E> [13:16:43] Request Try #3 / 5
<E> [13:16:43] Waiting 30 seconds before retrying...
<E> [13:17:13] Request Try #4 / 5
<E> [13:17:13] Waiting 40 seconds before retrying...
<E> [13:17:53] Request Try #5 / 5
<E> [13:17:54] Maximum number (5) of tries failed.

**Comment by ooggss** [comment]
请问你解决了吗，我也碰到了同样的问题，我的环境是ubuntu22 python3.9

**Comment by Fisher-Wang** [comment]
我也遇到了同样的问题，我的环境：

- ubuntu 20.04
- python 3.10
- bypy 1.8.2

**Comment by iamwangyabin** [comment]
我也遇到，但是每次都是重试了五次就开始上传了

**Comment by SunnyHaze** [comment]
同样的问题，ubuntu 20.04

**Comment by houtianze** [comment]
我刚刚试了一下，上传没遇到问题。

**Comment by houtianze** [comment]
可能是和#664是同一个原因：百度的秒传接口可能停了，还是改了。最新版`1.8.3`应该解决了这个问题，遇到`'error_code': 31023, 'error_msg': 'param error',`不会重试，不过秒传功能已经不可用。

**Comment by SunnyHaze** [comment]
谢谢作者大大！
